source/includes/steps-fle-convert-to-a-remote-master-key.yaml

```diff
@@ -52,22 +52,22 @@ content: |
            final BsonString awsAccessKeyId = new BsonString("<IAM User Access Key ID>");
            final BsonString awsSecretAccessKey = new BsonString("<IAM User Secret Access Key>");
            Map<String, Map<String, Object>> kmsProviders = new HashMap<String, Map<String, Object>>() {{
-             put("aws", new HashMap<String, Object>() {{
-               put("region", masterKeyRegion);
-               put("accessKeyId", awsAccessKeyId);
-               put("secretAccessKey", awsSecretAccessKey);
-             }});
+               put("aws", new HashMap<String, Object>() {{
```

Comment:
And here.  Once we make Java 9 the minimum we could use `java.util.Map#of()`, but most of our users are still on Java 8

Reply:
Will fix.